src/item-inspector/files-by-type/files-by-type-provider.ts

```diff
@@ -0,0 +1,43 @@
+import { html, TemplateResult } from 'lit-element';
+
+import './ia-files-by-type';
+import '@internetarchive/icon-share/icon-share';
+import {
+  IntMenuProvider,
+  IntProviderArgs,
+} from '../../interfaces/menu-interfaces';
+
+export class FilesByTypeProvider implements IntMenuProvider {
```

Comment:
QUESTION: What happened to adding JSDOC's to explain what the class does? e.g. I'm not sure what that letter "F" is doing in the `icon` property.

Reply:
Also curious about this class 👍

Reply:
Hello, pardon, this was my initial exploration on an "available theaters" menu. but then it started going deeper into design so i paused it.

This file nor <item-inspector> is part of this pull request.  This pr is scoped just for light refactor/code decoupling to be quickly consumed by bookreader & petabox. 👍